ownCloud/Client/ClientQueryViewController.swift

```diff
@@ -116,6 +117,16 @@ class ClientQueryViewController: UITableViewController, Themeable {
 		sortBar?.updateSortMethod()
 
         tableView.tableHeaderView = sortBar
+
+		searchController.searchResultsUpdater = self
+		searchController.obscuresBackgroundDuringPresentation = false
+		searchController.hidesNavigationBarDuringPresentation = true
+		searchController.searchBar.placeholder = "Search".localized
```

Comment:
Since the search bar in the "Files" app seems to do a global search, but this one doesn't, we should maybe rename this to "Search this folder" to make the scope of the search clearer.

Please also add that string to Localizable.strings.

Reply:
Definitely, I'll change it.